Aquality.Appium.Mobile/src/Aquality.Appium.Mobile/Elements/CheckBox.cs

```diff
@@ -4,44 +4,45 @@
 
 namespace Aquality.Appium.Mobile.Elements
 {
-    public class CheckBox : Element, ICheckBox
+    public class CheckBox : CheckableElement, ICheckBox
     {
         protected internal CheckBox(By locator, string name, ElementState state) : base(locator, name, state)
         {
         }
 
-        public bool IsChecked
+        public override bool IsChecked
         {
             get
             {
                 LogElementAction("loc.checkbox.get.state");
-                return GetElement().Selected;
+                return base.IsChecked;
```

Comment:
I think we olso need LogElementAction for RadioButtonElement
What if you add LogElementAction to base.IsChecked and put ElementType to message pattern?

Reply:
I believe we should not log this message for callings from other methods like "Toggle", "Check" and "Uncheck". 
There was a comment from @sunigos at java repo about it: https://github.com/aquality-automation/aquality-appium-mobile-java/pull/11#discussion_r395119700 .
Agree that we should probably log it for radiobutton

Reply:
Fixed